src/test/codegen/sanitizer_memtag_attr_check.rs

```diff
@@ -0,0 +1,13 @@
+// This tests that the sanitize_memtag attribute is
+// applied when enabling the memtag sanitizer.
+//
+// needs-sanitizer-memtag
+// only-aarch64
```

Comment:
`needs-sanitizer-memtag` should already cover this, no?

Reply:
Ah yea I believe you're right, this is redundant. I've removed this line.